frontend/src/lib/MlmdUtils.ts

```diff
@@ -49,10 +57,27 @@ export async function getTfxRunContext(argoWorkflowName: string): Promise<Contex
 /**
  * @throws error when network error, or not found
  */
-export async function getKfpRunContext(argoWorkflowName: string): Promise<Context> {
+async function getKfpRunContext(argoWorkflowName: string): Promise<Context> {
   return await getContext({ name: argoWorkflowName, type: 'KfpRun' });
 }
 
+async function getKfpV2RunContext(argoWorkflowName: string): Promise<Context> {
+  return await getContext({ name: argoWorkflowName, type: 'kfp.PipelineRun' });
+}
+
+export async function getRunContext(workflow: Workflow): Promise<Context> {
```

Comment:
How do you think about adding test for this function? Although it is simple enough which might not need test, I see that the order of context fetching can be something we want to test: `V2RunContext` surpass `TfxRunContext`, then if `TfxRunContext` not found, look up context with `kfpRunContext`.

Reply:
Sure, thanks for pushing for more tests! I'll add this now.